scripts/development/m3_stack/m3dbnode.yml

```diff
@@ -2,6 +2,15 @@ db:
   logging:
     level: info
 
+  tracing:
```

Comment:
hm are you going to leave this in? if so, why does the readme need the instructions for config updates?

Reply:
I changed the logic so that if it errors creating the tracer, we just log it and default to a noop - so it is safe to leave it in here.

Reply:
In case someone wants to change the sampling rate, etc.

Reply:
fair enough.

Reply:
> I changed the logic so that if it errors creating the tracer, we just log it and default to a noop - so it is safe to leave it in here.

I'm not sure I agree with that change; the most likely reason for that is invalid configuration, which it seems worthwhile to surface. Since it's analogous, do we handle failure to initialize tally etc in the same way? Would be good to be consistent.

Reply:
So I definitely log the error, but it would be kind of bad if m3db couldn't start if your jaeger host was down for some reason.

Reply:
Fair enough! My assumption was that the jaeger client would come up even if the host is down (and then just fail to log), and that it would error primarily in cases of misconfiguration. However, I could see the argument for not going down if jaeger for instance changes their configuration in a backwards incompatible way (though then the user upon upgrading will just have to notice that their traces are no longer working).

I'm good with this though; thanks for discussion!